spec/lib/salus/scanners/report_ruby_gems_spec.rb

```diff
@@ -2,6 +2,10 @@
 
 describe Salus::Scanners::ReportRubyGems do
```

Comment:
It would be helpful to also check that the end json can be validated against the cyclonedx schema (we did this for sarif).  
We tried briefly to find an existing function to do this, but couldn't find one.
If this is time consuming, then ignore the comment.

Reply:
I will look into this.